notify/opsgenie/opsgenie.go

```diff
@@ -171,10 +174,9 @@ func (n *Notifier) createRequests(ctx context.Context, as ...*types.Alert) ([]*h
 		}
 		requests = append(requests, req.WithContext(ctx))
 	default:
-		// https://docs.opsgenie.com/docs/alert-api - 130 characters meaning runes.
-		message, truncated := notify.TruncateInRunes(tmpl(n.conf.Message), 130)
+		message, truncated := notify.TruncateInRunes(tmpl(n.conf.Message), maxMessageLenRunes)
 		if truncated {
-			level.Debug(n.logger).Log("msg", "truncated message", "truncated_message", message, "alert", key)
+			level.Debug(n.logger).Log("msg", "Truncated message", "alert", key, "runes", maxMessageLenRunes)
```

Comment:
Standardized the capitalization of these to use a capital at the beginning. Please let me know if you'd rather me go the other way.